docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -409,138 +297,67 @@ EOF
 terraform apply
 ----
 
-. Create the first shown DNS records
-
-. Wait for the DNS records to propagate!
-+
-[source,console]
-----
-sleep 600
-host "api.${CLUSTER_ID}.${BASE_DOMAIN}"
-----
+. Set up DNS NS records on parent zone using the data from the Terraform output variable `dns_entries` from the previous step
 
-. Provision master nodes
+. Create LB hieradata
 +
-[source,console]
+[source,bash]
 ----
 cat > override.tf <<EOF
 module "cluster" {
-  bootstrap_count = 1
+  bootstrap_count = 0
+  master_count    = 0
   infra_count     = 0
   worker_count    = 0
 }
 EOF
-
-terraform apply
+terraform apply -target "module.cluster.local_file.lb_hieradata[0]"
 ----
```

Comment:
Provisioning the bootstrap node actually comes later.
Do the DNS records still get displayed even if `terraform apply -target "module.cluster.local_file.lb_hieradata[0]"` or should we just call `terraform apply` with the override file present ?

Reply:
@simu Do you have an input here?

Reply:
I don't understand the question / problem. The `lb_hieradata` resource should be able to be created without the bootstrap node without any issues.

Reply:
I have removed the bootstrap block and the `-target "module.cluster.local_file.lb_hieradata[0]"`.

Reply:
Please be aware that the bootstrap node may fail to initialize itself if the LBs are not present yet, as it won't have internet connectivity.

Reply:
Well, yes, that looks to be the same case as in the Exoscale guide. Do you have a suggestion in mind? There is a specific "Wait until LBs are fully initialized by Puppet" step before launching the bootstrap node.

Reply:
Why can't we leave it as `terraform apply -target "module.cluster.local_file.lb_hieradata[0]"`? I still don't understand the issue.

Reply:
I'm not terraform-fluent enough to know whether the DNS records still get displayed if we run `terraform apply -target "module.cluster.local_file.lb_hieradata[0]"` for the first time ever. Does this only run the hieradata-parts or already set up the loadbalancers too?

Reply:
The point is that this only sets up the hieradata, to ensure that the initial Puppet runs on the LBs will succeed.

Reply:
This is a non-issue if we create DNS records after creating the LBs.